README.md

```diff
@@ -59,7 +59,7 @@ iex> Litmus.validate(params, schema)
 
 ### Litmus.Type.String
 
-The `String` module contains options that will validate String data types. It supports the following options:
+The `String` module contains options that will validate String data types. It converts `boolean` and `number` data to type `string`. It supports the following options:
```

Comment:
I think we can change this addition to: `It converts boolean and number values to strings`